### ISSUE TITLE
CommandBar: A11y narrator doesn't read item as "selected"

### ISSUE THREAD

**Issue by ReeceKenney** [OfficeDev/office-ui-fabric-react#7288]
Using Narrator and Edge, when navigating the command bar items, it should read selected when you focus on the selected tab. At the moment it doesn't do this.

**Comment by betrue-final-final** [comment]
Can you provide a screenshot of the control in the bar you are referring to?

**Comment by ReeceKenney** [comment]
@betrue-final-final Sure! We have used the component in a calendar as a way to change views:
![image](https://user-images.githubusercontent.com/5323490/49649448-7a76a880-fa29-11e8-8b13-0fbf62ee00af.png)

**Comment by ReeceKenney** [comment]
@betrue-final-final Is Pivot a library of some sort? We are not using anything called Pivot.

Thanks!

**Comment by dzearing** [comment]
@ReeceKenney I'm super confused here. Your screenshot looks like you're using a `Pivot` component, documented here:

https://developer.microsoft.com/en-us/fabric#/components/pivot

But your title mentions `CommandBar` documented here:

https://developer.microsoft.com/en-us/fabric#/components/commandbar

Are you using the wrong component for the job? Or is the title wrong?
